### ISSUE TITLE
Toggle button to enable/disable filter

### ISSUE THREAD

**Issue by pascatl** [WhereGroup/spatial_filter#6]
It would be nice to have a button that enables/disables the currently selected filter.
I would have found it more intuitive if the button with the eye had already done this. In my opinion its functionality to show/hide the filter geometry isn't that important.

**Comment by kannes** [comment]
Agreed, this would make a nice addition. The current design requires one to 1) save the current filter, 2) remove it from the map/project and then, when wanted, 3) load it again from the saved filters. This was by design of our original funder.

We would probably happily accept a pull request with this change but I can't promise anything :)
